app/client/views/application/application.html

```diff
@@ -87,12 +87,12 @@
         <div class="field"
              ng-hide="isMitStudent">
           <p>
-            Because of limitations imposed by MIT, we are not legally allowed to host
-            non-MIT minors (those under 18) for HackMIT 2015. Checking the box below affirms that you are or will be 18 years or older by September 19th, 2015.
+            Because of limitations imposed by UCI, we are not legally allowed to host
+            non-UCI minors (those under 18) for HackUCI 2018. Checking the box below affirms that you are or will be 18 years or older by Feburary 2nd, 2018.
```

Comment:
Since we aren't accepting any minors period, this can be reworded "currently we aren't allowed to host minors" or something like that

Reply:
Also if we're making this field non-optional (mandatory) then you need to update a rule in applicationCtrl.js

Setup form function I think